### PR TITLE
FS Mailing Service

### PR DESCRIPTION
**Pull request by fszwedo**
Mailing service built using nodemailer.
To test it please add your mail in UnitImplementations/MailingServiceTest.ts 
and then run ```npm run start:mail```

You should receive an email shortly.

### CODE REVIEW

File: Api/Src/Services/MailingService.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+interface Message {
+    from: string,
+    to: string,
+    subject: string,
+    text: string,
+};
+
+interface Nodemailer {
+    createTransport({service, auth:{user, pass}})
```
Comment (kjreczek) [inline Api/Src/Services/MailingService.ts:9]: We could add @types/nodemailer library to our project for better typing, ofc it's just a suggestion.

**Review by kjreczek — COMMENTED** [review]
I don't see any major issues. It looks like it can be polished later so its fine.

File: Api/Src/Services/MailingService.ts (hunk 1 above)
Comment (fszwedo) [reply Api/Src/Services/MailingService.ts:9]: makes sense, I was trying to decouple the service from nodemailer itself but in the end I don't see any advantages of taking this approach - but as it's already done I'd keep it as it is, the current approach doesnt break anything and I don't need to redo the tests :)

**Review by kjreczek — APPROVED** [review]

**Comment by fszwedo** [comment]
@JustynaSmoliga could you please take a look so I could merge this one? :)

**Review by JustynaSmoliga — APPROVED** [review]
Code looks good, I don't see any major issues. I  received mail message and everything works.

**Pull request merged**